_author_instructions/01_policies.md

```diff
@@ -81,15 +82,9 @@ Articles can have any length; however, you should be as concise as possible.
 You are responsible for preparing your article PDF and materials in the way you want them to appear in LiveCoMS [using our templates](http://www.github.com/livecoms/author_templates), so it is important to take some care as to how your submitted materials look.
 LiveCoMS will not be separately typesetting your article for you, so be careful that your materials are laid out well.
 Be sure to pay particular attention to:
-- Place your figures and tables appropriately in the document, rather than collecting them at the end
-- Number figures and tables in the order in which they are referenced in the text
+- Place your figures and tables appropriately in the document (following the paragraph that first references them), rather than collecting them at the end. Do not not move the figures and tables in the LaTex document to change where they appear. Instead, use the [LaTex placement options](https://en.wikibooks.org/wiki/LaTeX/Floats,_Figures_and_Captions) to adjust positioning.
 - Use consistent (and legible) font sizes in figures and tables; our LaTeX template uses OpenSans; we recommend its use if possible in figures as well.
-
-### Representative image
```

Comment:
Couldn't figure out why you were removing this until I saw it was duplicated; again this must have gone wrong in a recent merge.